### PR TITLE
(Docs): Add service-account details in openebs-pool-pod-failure

### LINKED ISSUE #1150: (refactor): Add minimal permission & SvcAcc in openebs-pool-pod-failure exp
**What needs to be done**

- Add minimal permission and scope in openebs-pool-pod-failure chart - `chaos-chart` repo

- Add the service account details and chaos-engine manifest in openebs-pool-pod-failure docs - `litmus-docs` repo

### PR DESCRIPTION
**Pull request by ispeakc0de**
Signed-off-by: shubhamchaudhary <shubham.chaudhary@mayadata.io>

- Updating the service account sample in openebs-pool-pod-failure

- Update the sample engine manifest in openebs-pool-pod-failure

- Fixes: litmuschaos/litmus#1150

### CODE REVIEW

File: docs/openebs-pool-pod-failure.md [hunk 1]
```diff
@@ -64,9 +66,56 @@ If the experiment tunable DATA_PERSISTENCE is set to 'enabled':
...
+subjects:
+- kind: ServiceAccount
+  name: nginx-sa
+  namespace: default
+```
+
 ### Prepare ChaosEngine
 
 - Provide the application info in `spec.appinfo`
+- Provide the auxiliary applications info (ns & labels) in `spec.auxiliaryAppInfo`
```
Comment (uditgaurav) [inline docs/openebs-pool-pod-failure.md:118]: application's

**Review by ksatchit — DISMISSED** [review]

File: docs/openebs-pool-pod-failure.md [hunk 2]
```diff
@@ -64,9 +66,56 @@ If the experiment tunable DATA_PERSISTENCE is set to 'enabled':
...
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: nginx-sa
+  namespace: default
+  labels:
+    name: nginx-sa
+---
+# Source: openebs/templates/clusterrole.yaml
```
Comment (ksatchit) [inline docs/openebs-pool-pod-failure.md:84]: should we have this comment in its current form?

File: docs/openebs-pool-pod-failure.md [hunk 3]
```diff
@@ -87,12 +136,21 @@ metadata:
   name: target-chaos
   namespace: default
 spec:
+  # It can be app/infra
+  chaosType: 'infra' 
+  #ex. values: ns1:name=percona,ns2:run=nginx 
+  auxiliaryAppInfo: ""
   appinfo:
     appns: default
     applabel: 'app=percona'
```
Comment (ksatchit) [inline docs/openebs-pool-pod-failure.md]: nginx?

File: docs/openebs-pool-pod-failure.md (hunk 1 above)
Comment (ksatchit) [reply docs/openebs-pool-pod-failure.md:118]: we will do a bulk change for this!

**Review by ksatchit — APPROVED** [review]

**Pull request merged**